experimental/ComplexReflectionGroups/src/ComplexReflectionGroups.jl

```diff
@@ -0,0 +1,79 @@
+
+# A union type for all algebraic extensions of the field of rational numbers
+const QQAlgField = Union{NumField, QQField, QQBarField, QQAbField}
+const QQAlgFieldElem = Union{NumFieldElem, QQFieldElem, QQBarFieldElem, QQAbElem}
```

Comment:
```suggestion
const QQAlgFieldElem = Union{NumFieldElem, QQFieldElem, QQBarFieldElem, QQAbFieldElem}
```
has been renamed at some point